dbt/include/spark/macros/adapters.sql

```diff
@@ -168,11 +168,15 @@
   {%- endcall -%}
 {% endmacro %}
 
-{% macro spark__get_columns_in_relation(relation) -%}
-  {% call statement('get_columns_in_relation', fetch_result=True) %}
+{% macro spark__get_columns_in_relation_raw(relation) -%}
```

Comment:
@jtcohen6 Is it possible to make it dispatch pattern? We might want to override this.

Reply:
I submitted a PR #365, could you take a look at it? Thanks!

Reply:
Thank you @ueshin -- utilizing the dispatch pattern makes good sense. 👍